src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1152,3 +1166,146 @@ def _deny_request(
+    *,
+    connection_config: ConnectionConfig = Depends(_get_connection_config),
+    privacy_request_id: str,
+    db: Session = Depends(deps.get_db),
+) -> Optional[Dict[str, Any]]:
+    """
+    View uploaded data for this privacy request for the given access manual webhook
+    """
+    privacy_request: PrivacyRequest = get_privacy_request_or_error(
+        db, privacy_request_id
+    )
+    access_manual_webhook: AccessManualWebhook = get_access_manual_webhook_or_404(
+        connection_config
+    )
+
+    if not privacy_request.status == PrivacyRequestStatus.requires_input:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST,
+            detail=f"Invalid access manual webhook upload request: privacy request '{privacy_request.id}' status = {privacy_request.status.value}.",  # type: ignore
+        )
+
+    try:
+        logger.info(
+            "Retrieving input data for access manual webhook '%s' for privacy request '%s'.",
+            connection_config.key,
+            privacy_request.id,
+        )
+        return privacy_request.get_manual_webhook_input(access_manual_webhook)
+    except MissingManualWebhookData as exc:
+        logger.info(exc)
+        return access_manual_webhook.empty_fields_dict
+    except PydanticValidationError:
+        raise HTTPException(
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=f"Saved fields differ from fields specified on webhook '{access_manual_webhook.connection_config.key}'. "
+            f"Re-upload manual data using '{PRIVACY_REQUEST_ACCESS_MANUAL_WEBHOOK_INPUT.format(connection_key=connection_config.key, privacy_request_id=privacy_request.id)}'.",
+        )
+
+
+@router.post(
+    PRIVACY_REQUEST_RESUME_FROM_REQUIRES_INPUT,
+    status_code=HTTP_200_OK,
+    response_model=PrivacyRequestResponse,
+    dependencies=[
+        Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_CALLBACK_RESUME])
+    ],
+)
+async def resume_privacy_request_from_requires_input(
+    privacy_request_id: str,
+    *,
+    db: Session = Depends(deps.get_db),
+) -> PrivacyRequestResponse:
+    """Resume a privacy request from 'requires_input' status."""
+    privacy_request: PrivacyRequest = get_privacy_request_or_error(
+        db, privacy_request_id
+    )
+
+    if privacy_request.status != PrivacyRequestStatus.requires_input:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST,
+            detail=f"Cannot resume privacy request from 'requires_input': privacy request '{privacy_request.id}' status = {privacy_request.status.value}.",  # type: ignore
+        )
+
+    access_manual_webhooks: List[AccessManualWebhook] = AccessManualWebhook.get_enabled(
+        db
+    )
+    try:
+        for manual_webhook in access_manual_webhooks:
+            privacy_request.get_manual_webhook_input(manual_webhook)
+    except (MissingManualWebhookData, PydanticValidationError) as exc:
+        raise HTTPException(
```

Comment:
does this mean we don't allow for partial uploads? For example- one upload request for email, and another for address?

Reply:
Discussed in slack, but we just need confirmation that data was queried for all manual webhooks.  It's okay if nothing was found; all fields can be None, but there needs to be an entry for each manual webhook in the cache.